### ISSUE TITLE
Fix Bitcoin Book Srapper

### ISSUE THREAD

**Issue by elraphty** [bitcoinsearch/scraper#84]
Currently the bitcoin book scrapper does not work, the urls do not exists and beautiful soup cannot scrape the chapters of the book.

- Write a fix to enable the bitcoin book scrapper work with the new version of the book on Github
- Update the readme with the changes

**Comment by kouloumos** [comment]
Good point. As you can see in [sources](https://bitcoinsearch.xyz/sources), we last scraped the Bitcoin Book (Mastering Bitcoin) a year ago. Normally, this wouldn’t be an issue since books aren’t frequently updated, but the new edition has been released since then.

The chapters were indexed as separate documents during the last scrape (https://github.com/bitcoinsearch/scraper/issues/46), which aligns with the chunking concept we've been discussing. While we successfully broke the content into chunks, there isn’t yet a system in place to connect these chunks meaningfully. If we decide to adopt the chunking strategy moving forward, this would be a good opportunity to implement it effectively for the book as well.
